server/backbone/api/helpers.py

```diff
@@ -181,3 +183,24 @@ def get_childs_quest_with_window(start, lookahead):
                            start <= find_next_time(q, start) <= end]
 
     return one_time_relevants + recurring_relevants
+
+
+def _until_next_level(current_level):
+    # For now this is a constant 12 xp / level, but this
+    #  function makes us able to change this on the fly
+    str(current_level)  # suppress warning about unused parameter
```

Comment:
?

Reply:
I wanted to keep this on the server, so we can change how much xp is needed to level up on the fly without updating the app, but for now I kept the xp a flat 12xp / level.

Reply:
The `str()` call just stops pycharm from complaining about unused arguments